packages/kbn-discover-utils/src/components/custom_control_columns/types.ts

```diff
@@ -22,6 +22,7 @@ export interface RowControlProps {
   label: string;
   iconType: IconType;
   onClick: ((props: RowControlRowProps) => void) | undefined;
+  tooltipContent: React.ReactNode;
```

Comment:
```suggestion
  tooltipContent?: React.ReactNode;
```
Should this be optional?